packages/vscode-dependencies-validation/test/autofix/unsupportedFilesWatcher.spec.ts

```diff
@@ -0,0 +1,120 @@
+import type {
```

Comment:
I am not diving into this test in-depth right now.
I think this has the same questions of blackbox vs whitebox unit tests + mocks
and what are the alternatives when we are trying to test "events" / "mini flows"